app/models/solr_document.rb

```diff
@@ -40,6 +55,23 @@ def collection?
     level == self.class::COLLECTION_LEVEL
   end
 
+  def component?
+    key?("component_level_isim")
+  end
+
+  def html_presenter_class
```

Comment:
It looks like the implementation of both of these are the same - why do we need two presenters?